docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -1,72 +1,96 @@
-For example, if you plan to access the BigchainDB API over HTTP, you might
-not need the ``https-certs`` Secret.
-   
-
-Ensure that all the secrets are base64 encoded values and the unused ones
-are set to an empty string.
-For example, assuming that the public key chain is named ``cert.pem`` and
-private key is ``cert.key``, run the following commands to encode the
-certificates into single continuous string that can be embedded in yaml,
-and then copy the contents of ``cert.pem.b64`` in the ``cert.pem`` field,
-and the contents of ``cert.key.b64`` in the ``cert.key`` field. 
-      
+How to Configure Your BigchainDB Node
+=====================================
+
+This page outlines the steps to set a bunch of configuration settings
+in your BigchainDB node.
+They are pushed to the Kubernetes cluster in two files,
+named ``config-map.yaml`` (a set of ConfigMaps)
+and ``secret.yaml`` (a set of Secrets).
+They are stored in the Kubernetes cluster's key-value store (etcd).
+
+Make sure you did all the things listed in the section titled
+:ref:`Things Each Node Operator Must Do`
+(including generation of all the SSL certificates needed
+for MongoDB auth).
+
+
+Edit config-map.yaml
+--------------------
+
+Make a copy of the file ``k8s/configuration/config-map.yaml``
+and edit the data values in the various ConfigMaps.
+That file already contains many comments to help you
+understand each data value, but we make some additional
+remarks on some of the values below.
+
+vars
+~~~~
+
+Your BigchainDB cluster organization should have a standard way
+of naming instances, so the instances in your BigchainDB node
+should conform to that standard (i.e. you can't just make up some names).
+There are some things worth noting about the ``mdb-instance-name``:
+
+* MongoDB reads the local ``/etc/hosts`` file while bootstrapping a replica
+  set to resolve the hostname provided to the ``rs.initiate()`` command.
+  It needs to ensure that the replica set is being initialized in the same
+  instance where the MongoDB instance is running.
+* We use the value in the ``mdb-instance-name`` field to achieve this.
+* This field will be the DNS name of your MongoDB instance, and Kubernetes
+  maps this name to its internal DNS.
+* This field will also be used by other MongoDB instances when forming a
+  MongoDB replica set.
+* We use ``mdb-instance-0``, ``mdb-instance-1`` and so on in our
+  documentation. Your BigchainDB cluster may use a different naming convention.
+
+bdb-keyring.bdb-keyring
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This lists the BigchainDB public keys
+of all *other* nodes in your BigchainDB cluster
+(not including the public key of your BigchainDB node). Cases:
+
+* If you're deploying the first node in the cluster,
+  the value should be ``""`` (an empty string).
+* If you're deploying the second node in the cluster,
+  the value should be one public key inside double quotes
+  (*not* base64-encoded). For example,
```

Comment:
Okay, I'll make a top-level remark about that for `config-map.yaml`, rather than per data value.